docs/wcf-config.md

```diff
@@ -8,16 +8,31 @@ Example project available in
 [test/test-applications/integrations/TestApplication.Wcf.Client.NetFramework](../test/test-applications/integrations/TestApplication.Wcf.Client.NetFramework/)
 folder.
 
-⚠️ **NOTICE:** Instrumentation of
-[APM-style](https://learn.microsoft.com/en-us/dotnet/standard/asynchronous-programming-patterns/asynchronous-programming-model-apm)
-calls is supported, but have known limitations.
-It is recommended to convert them to
-[TAP-style](https://learn.microsoft.com/en-us/dotnet/standard/asynchronous-programming-patterns/task-based-asynchronous-pattern-tap)
-calls.
-
 ## WCF Client Configuration (.NET)
 
-Instrumentation for WCF Client on .NET is not supported.
+Add `OpenTelemetry.Instrumentation.Wcf` and `System.Diagnostics.DiagnosticSource`
+package to the project. The version of `OpenTelemetry.Instrumentation.Wcf`
+should match the one used by AutoInstrumentation library to avoid compatibility
+issues.
```

Comment:
Is there any way to avoid manual steps?
If so, we can wait with introducing it post GA. This version is not autoinstrumentation friendly.